feeadjuster/feeadjuster.py

```diff
@@ -21,7 +21,44 @@ def get_ratio(our_percentage):
     return 50**(0.5 - our_percentage)
 
 
+def get_ratio_hard(our_percentage):
+    """
+    Return value is between 0 and 20: 0 -> 20; 0.5 -> 1; 1 -> 0
+    Between 0 and 0.5 is exponential and between 0.5 and 1 is linear
+    """
+    if our_percentage > 0.5:
+        return (1 - our_percentage) * 2
+    return 400**(0.5 - our_percentage)
```

Comment:
I wanted a function where f(0)=20, f(0.5)=1 and f(1)=0, and this was my first idea.
I'll write a simpler version.

Reply:
I just thought if we might also need a check that prevents/limits setting too high or low values anyway

Reply:
(regardless of what method/function is used)

Reply:
If I have no liquidity in a channel, I want to set up a very high fee for that. It is a message for the other nodes: "please do not use that channel (because it will fail anyway if you try)".

Reply:
@gallizoltan  I am more concerned if tracking `plugin.adj_balances[in_scid]["our"]` of channel balance is reliable. If not we can run into negative or infinity fee values:
https://github.com/lightningd/plugins/blob/d55b9a273bf9b6b699413e4add0c5747290b6d50/feeadjuster/feeadjuster.py#L132-L133

Reply:
What if we would rewrite the forward event? 
```python
@plugin.subscribe("forward_event")
def forward_event(plugin: Plugin, forward_event: dict, **kwargs):
    time.sleep(random.randint(1, 5)) # give some time for the changes to take place
    feeadjust(plugin)
```

Reply:
No, please not like this. @darosior said in IRC the calculation is safe as we only watch for 'settled' HTLC events.
In this case you can ignore my remark. Anyway we should add a simple check:  `fees >= 0` to prevent crashes incacse we are wrong.

Reply:
On the upper end we could check for 100 times base fee. not sure if this is perfect though. Your choice ;)